tutorials/tutorial14_rllib_ec2.ipynb

```diff
@@ -0,0 +1,158 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 14: Running RLlib experiments on EC2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial walks through how to run RLlib experiments on an AWS EC2 instance. This assumes that the machine you are using has already been configured for AWS (i.e. `~/.aws/credentials` is properly set up). We HIGHLY RECOMMEND the following as prior reading, as this will be something of an abridged version: https://github.com/ray-project/ray/blob/master/doc/source/autoscaling.rst\n",
+    "\n",
+    "\n",
+    "While going through the above documentation, you can ignore all instructions on GCP, as GCP will not be covered in this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## A brief description of ray_autoscale.yaml\n",
+    "\n",
+    "This section explains the most salient components of `/flow/scripts/ray_autoscale.yaml`. We'll go over some of the variables you should change, as well as those that might come in handy for you. A more detailed guide is on deck soon.\n",
+    "\n",
+    "* `cluster_name`: (CHANGE ME!) A unique identifier for the head node and workers of this cluster. If you want to set up multiple clusters, `cluster_name` must be changed each time the script is run.\n",
+    "* `setup_commands`: This describes the set of commands to run after the instance is up and running in the $HOME directory. Commands can vary widely. If you're running experiments, you're most likely on a branch that is not 'master'. This is the right place to specify which branch you want to sync to EC2. \n",
+    "    * To specify a branch from the main flow-project repo will look something like: \n",
+    "    \n",
+    "    `cd flow && git pull && git checkout [YOUR-BRANCH-HERE]`\n",
+    "    * To specify a branch from your fork, the command will look something like this:\n",
+    "    \n",
+    "    `git remote add https://github.com/[USER]/flow && git fetch [USER] && git checkout [USER] [YOUR-BRANCH-HERE]\n",
```

Comment:
think this should be git remote add [USER] https://github.com/[USER]/flow or else their branch name is undefined